src/unity/schemas/OperatorUser.yml

```diff
@@ -0,0 +1,23 @@
+properties:
+  email:
+    type: string
+    description: user's email
+  firstName:
+    type: string
+    description: user's first name
+  id:
+    type: string
+    description: user's id in quartz
+  idpeId:
+    type: string
+    description: user's id in idpe
+  lastName:
+    type: string
+    description: user's last name
+  operator:
+    type: boolean
+    description: whether or not the user is an operator
+  sfdcContactId:
```

Comment:
Wow! I'm pretty sure this field doesn't exist any more, but that's out of scope for this PR.

Reply:
It is actually still being referenced by the UI

Reply:
https://github.com/influxdata/ui/blob/2519757d1dcc2dc2163a313e784ed41a5722d7c2/src/operator/account/AccountGrid.tsx#L68

Reply:
I'd be pretty surprised if it ever shows anything but `N/A`.